### DISCUSSION TITLE
169

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#623] [Comments]
http://jojoldu.tistory.com/169

**Comment by jojoldu** [comment]
작성자: seungdols

저도 맥북 쓰느라 DBMS는 외부에 설치하고 싶었는데 ㅎㅎ 마침 도커 생각이 났는데 딱 정리를 해주셨네요!!ㅎㅎ
감사합니다.

**Comment by jojoldu** [comment]
작성자: ㅇㅣ게

제일좋은거같네요

**Comment by jojoldu** [comment]
작성자: S.ONg.c

감사합니다 잘 보고 잘 따라 해서 설치 하고 연결 까지는 되는데 혹시 일반 커맨드처럼 터미널에서 sqlplus 문은 실행 따로 못하는 지요?? 궁굼합니다.

**Comment by jojoldu** [comment]
작성자: Inor

감사합니다. 덕분에 맥북에서 오라클을 사용할 수 있게 됐습니다. 혹시 괜찮으면 위 포스팅을 따라하며 겪었던 문제점들을 추가로해서 내용을 공유해도 되는지 알고싶습니다 ~ 출처는 꼭 남기겠습니다!

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

아 넵넵 출처만 명확히 남겨주시면 괜찮습니당!

**Comment by jojoldu** [comment]
작성자: Inor

도커를 이용해서 오라클을 사용하는 과정에 궁금한 점이 생겼는데 혹시 질문 가능할까요?

system/oracle로 접근해서 새로운 유저를 생성했는데 도커를 껏다가 다시 켜면 유저가 사라져 있습니다. 혹시 도커를 재시작하면 오라클에 설정한 테이블이나 스키마가 영구적으로 저장이 불가능한 구조인가요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

저도 도커 고수는 아니지만 기본적으로는 안되는걸로 알고 있습니다.

아래 링크를 참고하셔서 로컬PC에 Database에서 영구보관해야할 부분을 별도로 관리하도록 하는 방법을 참고하시면 될것 같습니다.

http://arisu1000.tistory.com/27809

**Comment by jojoldu** [comment]
작성자: Inor

감사합니다!! 덕분에 볼륨이라는 개념을 알게됐고 해결 방법을 찾았습니다!! 간단하게 사용할 수 있는줄 알았지만 역시 끝이 없군요 ㅋㅋㅋㅋ덕분에 도커 사용법도 공부하고 !! 감사합니다 ~ 자주 찾아 뵙겠습니다 ~

https://github.com/wnameless/docker-oracle-xe-11g/issues/64

이곳에서 찾았습니다 ~

**Comment by jojoldu** [comment]
작성자: csii

감사합니다. 글을 보고 따라하여 실행까지 다 되었는데요 docker run -p 59160:22 명령어는 어떤것을 의미하는건가요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

아! 설명이 누락되었네요^^;
해당 Docker 컨테이너로 SSH 접속(PORT:22) 을 위한 포트입니다.

https://stackoverflow.com/questions/28134239/how-to-ssh-into-docker

**Comment by jojoldu** [comment]
작성자: csii

감사합니다!!

**Comment by jjjjjjeonda86** [comment]
intellij를 다운 받았는데 저 설명해주시는 화면으로 가려면 어떻게 해야 하나요ㅠㅠ

**Comment by leedh** [comment]
글 감사합니다.
근데 이 글 보고 따라했는데, 네트워크 에러가 계속나서 헤매다가

SSH server has been removed since 18.04, please use "docker exec" or 16.04 instead.

라고 하더라구요
(https://hub.docker.com/r/wnameless/oracle-xe-11g/)

> docker pull wnameless/oracle-xe-11g:16.04
로 해야할 것 같습니다.

**Comment by jojoldu** [comment]
@leedh 앗? 첨언 감사합니다 :)

**Comment by jojoldu** [comment]
@jjjjjjeonda86 IntelliJ 유료버전에서 database를 찾으시면 됩니다!
무료버전에선 지원하지 않습니다 ㅠ
